torch/csrc/api/src/jit.cpp

```diff
@@ -0,0 +1,19 @@
+#include <torch/jit.h>
+
+#include <torch/csrc/jit/script/compiler.h>
+#include <torch/csrc/jit/stack.h>
+
+#include <memory>
+#include <string>
+
+namespace torch {
+namespace jit {
+
+std::shared_ptr<script::Module> compile(const std::string& source) {
+  auto module = std::make_shared<script::Module>();
+  defineMethodsInModule(*module, source, script::nativeResolver, nullptr);
```

Comment:
Tip: For functions that take many arguments, and especially for values that don't mean anything when read, like `nullptr`, it's good practice to use what we call "LLVM-style comments" by writing `/*name_of_argument=*/value` so that the reader has a better idea of what is going on without having to consult the function declaration